gossipd/test/run-bench-find_route.c

```diff
@@ -277,6 +278,13 @@ int main(int argc, char *argv[])
 		if (route_lengths[i])
 			printf(" Length %zu: %zu\n", i, route_lengths[i]);
 
+	/* Since we omitted destructors on these, clean up manually */
```

Comment:
Maybe we can afford 40 bytes to write a destructor for the entire `rstate` that does `free_chan` on each `chan` it owns, instead of duplicating this code at each test?

Reply:
I feel like an idiot; that is much neater!  Thanks!